index.html

```diff
@@ -61,7 +61,7 @@ <h3>Canopy</h3>
 				</div>
 			</div>
 			<div class="card white-bg reverse">
```

Comment:
- [ ] [REQUIRED] For the cards in the work section, there ought to be some margin by the sides. Kindly implement them. 

### Your Implementation
![Screenshot from 2022-03-25 19-57-01](https://user-images.githubusercontent.com/58692590/160187002-420d92e8-572c-4a9a-90a0-819561e9c0ef.png)

### Figma Design
![Screenshot from 2022-03-25 19-57-08](https://user-images.githubusercontent.com/58692590/160187237-e64c1eb6-4e37-4c5b-8f58-a37788fff1e7.png)